whist/server/api/game/trick.py

```diff
@@ -85,4 +95,5 @@ def get_winner(game_id: str, user: Player = Depends(get_current_user),
         winner = trick.winner
     except TrickNotDoneWarning:
         return {'status': 'The trick is not yet done, so there is no winner.'}
+    logger.info(user.username + " has won")
```

Comment:
This is not correct here. The winner is not `user` but `winner`.

I'm not if this route really needs logging, but if we do logging here, I would log the `user` who requested the winner information.